scripts/release_notes/classify_prs.py

```diff
@@ -119,3 +125,10 @@ def format_prs(mod_df):
 # Missing PRs are these ones... classify them manually
 missing_prs = pd.concat([mod_df, included_prs]).drop_duplicates(subset="pr_number", keep=False)
 print(format_prs(missing_prs))
+
+# In[12]:
+
+# Generate list of authors
+command_to_run = f"{{ git shortlog -s {previous_release}..{current_release} | cut -f2- & git log -s {previous_release}..{current_release} | grep Co-authored | cut -f2- -d: | cut -f1 -d\< | sed 's/^ *//;s/ *$//' ; }} | sort --ignore-case | uniq | tr '\n' ';' | sed 's/;/, /g;s/, $//' | fold -s"
```

Comment:
The [linter](https://app.circleci.com/pipelines/github/pytorch/vision/14980/workflows/a131a866-e4fb-4423-a2f4-26c4a0310ea4/jobs/1206694) is complaining about this line. It might be due to a change made yesterday to run temporarily on root. This was patched today so let's merge main and retry.